workflow-templates/pr-automerge-open-release.yml

```diff
@@ -0,0 +1,67 @@
+# For non-draft changes to Named Release branches:
+# - Check if the user belongs to a maintainers team.
+# - If so, approve the pull request.
+#   - Tag community-engineering (for now) and the maintainers team.
+#   - Merge the PR when the author comments `@edx-community-bot merge`.
+# Required organization secrets
+# - CC_GITHUB_TOKEN=...
+# - CC_TEAM_CHAMPIONS=org/team-name
+# - CC_TEAM_CONTRIBUTORS_ORG=org
+# - CC_TEAM_CONTRIBUTORS_TEAM=team-name
+---
+name: automerge BTR open-release PRs
+on:
+  issue_comment:
+    branches:
+    - open-release/*.master
+    types:
+    - created
+    - edited
+  pull_request_target:
+    branches:
+    - open-release/*.master
```

Comment:
why not `open-release/*`? Shouldn't CCs have access to .rc branches as well?

Reply:
@sarina Is that officially part of the process? Repos I spot-checked did not often have `.rc` branches.

And in particular, places that did, things like `ficus.rc` or more often `ficus.1`/`ficus.2` were _tags_ not _branches, so they didn't support merging.

@nedbat Can you clarify the expected branching patterns?

Reply:
We have sometimes used branches, but as "movable tags" rather than as distinct from the .master branches.  But I think "open-release/*" makes sense just to remain flexible into the future without having to revisit the pattern here later.